mods/ra/weapons/explosions.yaml

```diff
@@ -212,6 +212,14 @@ UnitExplodeSmall:
 		SmudgeType: Crater
 		InvalidTargets: Structure, Wall, Trees
 
+CivBuildingExplosion:
```

Comment:
Can you explain what the purpose of this and the Explodes trait on the default is?